src/psyclone/generator.py

```diff
@@ -445,7 +445,7 @@ def write_unicode_file(contents, filename):
         if not isinstance(contents, unicode):
             contents = unicode(contents, 'utf-8')
         # pylint: enable=undefined-variable
-        encoding = {}
+        encoding = {'encoding': 'utf-8'}
```

Comment:
Since the `encoding = ` assignment is now duplicated, please could you move it to just before the call to `io.open`.

Reply:
Requested changes made as 63476a2